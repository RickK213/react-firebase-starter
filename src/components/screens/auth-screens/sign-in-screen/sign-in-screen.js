import React from 'react';
import { Link } from 'react-router-dom';
import { SignInForm } from './sign-in-form/sign-in-form';
import { ROUTES } from '../../../../constants/routes';

export const SignInScreen = () => {
  return (
    <div>
      <h2>Sign In</h2>
      <SignInForm />
      <p>
        <Link to={ROUTES.PASSWORD_FORGET.path}>
          {ROUTES.PASSWORD_FORGET.label}
        </Link>
      </p>
    </div>
  );
};
