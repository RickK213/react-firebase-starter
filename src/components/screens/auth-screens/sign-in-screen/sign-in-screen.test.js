/* global React, create */
import { StaticRouter } from 'react-router-dom';
import { SignInScreen } from './sign-in-screen';

describe('<SignInScreen />', () => {
  it('should render correctly', () => {
    const component = create(
      <StaticRouter context={{}}>
        <SignInScreen />
      </StaticRouter>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
