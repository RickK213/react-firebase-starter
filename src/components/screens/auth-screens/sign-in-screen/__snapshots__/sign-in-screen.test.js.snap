// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignInScreen /> should render correctly 1`] = `
<div>
  <h2>
    Sign In
  </h2>
  <form
    autoComplete="off"
    onSubmit={[Function]}
    style={
      Object {
        "width": "360px",
      }
    }
  >
    <input
      autoComplete="off"
      name="email"
      onChange={[Function]}
      placeholder="Email Address"
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
          "padding": ".5rem",
          "width": "100%",
        }
      }
      type="email"
      value=""
    />
    <input
      autoComplete="off"
      name="password"
      onChange={[Function]}
      placeholder="Password"
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
          "padding": ".5rem",
          "width": "100%",
        }
      }
      type="password"
      value=""
    />
    <button
      disabled={true}
      style={
        Object {
          "padding": ".5rem",
          "textTransform": "uppercase",
        }
      }
      type="submit"
    >
      Sign In
    </button>
  </form>
  <p>
    <a
      href="/account/password-forget"
      onClick={[Function]}
    >
      Forgot Password?
    </a>
  </p>
</div>
`;
